surfnperf.js

```diff
@@ -356,6 +356,56 @@
     return this.duration('navigationStart', 'loadEventEnd');
   };
 
+  /**
+   * Time to first paint (browser-reported time for the first pixels to be rendered on the page)
+   *
+   * @returns {integer} time in ms
+   * @memberOf SurfNPerf
+   */
+  SNPProto.getFirstPaint = function() {
+    if(this.chromeLoadTimes()) {
+      return this.chromeLoadTimes().firstPaintTime * 1000 - this.navigationStartTiming();
+    } else if(window.performance && window.performance.timing && this.msFirstPaint()) {
+      return this.msFirstPaint() - this.navigationStartTiming();
```

Comment:
use `SurfNPerf.performanceTiming().navigationStart` instead of `this.navigationStartTiming()`